Makefile

```diff
@@ -19,3 +22,9 @@ uninstall:
 	$(MAKE) -C rune uninstall
 	$(MAKE) -C runectl uninstall
 	$(MAKE) -C shim uninstall
+
+rpm:
+	@mkdir -p /root/rpmbuild/{BUILD,RPMS,SOURCES,SPECS,SRPMS}
```

Comment:
Don't do this. If current user is not root or not in a container, then fails for sure.

Reply:
Use mktemp command to create a dir used to hold rpmbuild.